### PR TITLE
Dao-refactoring

### PR DESCRIPTION
**Pull request by shivamgarglab49**
Signed-off-by: shivam_garg <shivam.garg@iongroup.com>

### CODE REVIEW

File: e2e-test/business/Common.ts [hunk 1]
```diff
@@ -273,13 +273,15 @@ export default class Common extends Base {
     const response = await new TokenInfoQuery()
       .setTokenId(tokenId)
       .execute(client);
+    const isNFT = response.tokenType! === TokenType.NonFungibleUnique;
```
Comment (amandeeprehal-lab49) [inline e2e-test/business/Common.ts:276]: Minor - Pls add space between  response.tokenType !=== TokenType.NonFungibleUnique

File: e2e-test/business/Common.ts [hunk 2]
```diff
@@ -380,6 +382,43 @@ export default class Common extends Base {
...
+  static transferAssets = async (
+    tokenId: string | TokenId,
+    amountOrId: number,
+    toAccountId: string | AccountId,
+    fromAccountId: string | AccountId,
+    fromPrivateKey: PrivateKey,
+    client: Client = clientsInfo.operatorClient
+  ) => {
+    const txn = new TransferTransaction();
+    if (
```
Comment (amandeeprehal-lab49) [inline e2e-test/business/Common.ts:394]: Suggestion - Can you add Switch statement here, it will add more readability?

File: e2e-test/business/TokenTransferGovernor.ts [hunk 3]
```diff
@@ -84,15 +82,13 @@ export default class TokenTransferGovernor extends Governor {
     client: Client = clientsInfo.operatorClient,
     description: string = this.DEFAULT_DESCRIPTION,
     link: string = this.DEFAULT_LINK,
-    nftTokenSerialId: number = this.DEFAULT_NFT_TOKEN_SERIAL_NO,
-    creator: string = clientsInfo.operatorId.toSolidityAddress()
+    nftTokenSerialId: number = this.DEFAULT_NFT_TOKEN_SERIAL_NO
```
Comment (amandeeprehal-lab49) [inline e2e-test/business/TokenTransferGovernor.ts:85]: I think, you missed line 63 to remove creator

File: e2e-test/step-definitions/FTDaoFactoryTest.ts [hunk 4]
```diff
@@ -130,7 +129,7 @@ export class FTDaoFactoryTest extends CommonSteps {
     description: string,
     link: string
   ): Promise<void> {
-    proposalId = await ftDao.createTokenAssociateProposal(
+    proposalId = await governor.createTokenAssociateProposal(
```
Comment (amandeeprehal-lab49) [inline e2e-test/step-definitions/FTDaoFactoryTest.ts:132]: pls clean up import

File: integrationTest/governance/governorUpgrade.ts
Comment (amandeeprehal-lab49) [inline integrationTest/governance/governorUpgrade.ts:1]: We should also integration tests where we transfer NFT token while NFT being GOD token. May be in separate PR.

Comment (shivamgarglab49) [reply integrationTest/governance/governorUpgrade.ts:1]: sure

**Review by amandeeprehal-lab49 — APPROVED** [review]
